### ISSUE TITLE
Warn on binding to <input>'s value attribute.

### ISSUE THREAD

**Issue by justinfagnani** [43081j/eslint-plugin-lit#20]
Binding to <input>'s value attribute is often an error, causing confusion then subsequent updates to the bound expression fail to update the input element. Most often the author should be binding to the value property.

So warn on:

```js
html`<input value=${x}>`
```

And suggest a fix of:

```js
html`<input .value=${x}>`
```
